main.go

```diff
@@ -221,6 +221,8 @@ func Flash(pkgName, port string, options *compileopts.Options) error {
 			fileExt = ".bin"
 		case strings.Contains(config.Target.FlashCommand, "{uf2}"):
 			fileExt = ".uf2"
+		case strings.Contains(config.Target.FlashCommand, "{zip}"):
```

Comment:
The name zip is probably too common and will cause problems later.
I think the name `nrf-dfu`, as written by @aykevl , is fine.